e2e/support/helpers/e2e-misc-helpers.js

```diff
@@ -306,7 +306,7 @@ export function saveQuestion(
   { wrapId = false, idAlias = "questionId" } = {},
 ) {
   cy.intercept("POST", "/api/card").as("saveQuestion");
-  cy.findByText("Save").click();
+  cy.findByTestId("qb-header").button("Save").click();
```

Comment:
I was just dealing with this helper and have thought about this change.
Thanks for improving it.